index.html

```diff
@@ -0,0 +1,1741 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
```

Comment:
- [ ] You need to install Bootstrap in your project with any means mentioned in their webpage [here](https://getbootstrap.com/docs/4.5/getting-started/introduction/). Downloading the whole CSS file increases the size of your project and could impact any future changes you want to make to it. I couldn't find this suggestion on the Bootstrap page, if you have any sources instructing you to do what you did, can you please send it to me?